src/components/LocationDetails.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ReactTooltip from "react-tooltip";
+import Collapsible from "react-collapsible";
+
+function LocationDetails(props) {
+  const featureProps = props.feature.properties;
+
+  const propsMapping = [
+    {
+      title: 'Demographic Summary',
+      content: [
+        { id: 'persons_num', label: 'Population' },
+        { id: 'median_aud', label: 'Median Income', format: (val) => `AUD ${val}` },
```

Comment:
I'd prefer to stick with toLocaleString() for formatting currency as is currently done for [sa2_properties in map.js](https://github.com/CxSci/SA-dashboard/blob/master/src/components/map.js#L401-L404), as this project may be adapted in the future for other countries and other currencies, and it'll be easier if we don't hardcode formats.